src/App.jsx

```diff
@@ -14,57 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setSelectedGood('');
+            }}
+          />
+        )}
```

Comment:
Here you can combine these two checks into one ternary:

```suggestion
             {selectedGood ? (
          <>
            {`${selectedGood} is selected`}
            <button
              data-cy="ClearButton"
              type="button"
              className="delete ml-3"
              onClick={() => {
              setSelectedGood('');
            }}
            />
          </>
        ) : (
          'No goods selected'
        )}
```

Reply:
doesn't it hurt readability?

Reply:
Maybe you are right, let's wait for the mentor's answer

Reply:
> Maybe you are right, let's wait for the mentor's answer

It's not only about readability in this case, you have two options and render different UI for them. If you decide to move the UI for the selected good to a separate component you will need to control it in two places